exporter/pbm_collector.go

```diff
@@ -112,7 +95,7 @@ func (p *pbmCollector) collect(ch chan<- prometheus.Metric) {
 
 	pbmConfig, err := pbmClient.GetConfig(p.ctx)
 	if err != nil {
-		logger.Errorf("failed to get PBM configuration: %s", err.Error())
+		logger.Warnf("failed to get PBM configuration: %s", err.Error())
```

Comment:
I think it should be Info level if we expect that pbm config might not exist there